clients/da-vinci-client/src/main/java/com/linkedin/davinci/listener/response/ReadResponse.java

```diff
@@ -19,8 +18,8 @@ public abstract class ReadResponse {
   private int multiChunkLargeValueCount = 0;
   private CompressionStrategy compressionStrategy = CompressionStrategy.NO_OP;
   private boolean isStreamingResponse = false;
-  private Optional<List<Integer>> optionalKeySizeList = Optional.empty();
-  private Optional<List<Integer>> optionalValueSizeList = Optional.empty();
+  private List<Integer> keyListSize;
+  private List<Integer> valueListSize;
```

Comment:
Thanks. Good to change. Can we use a fast-util `IntList` instead of Java's boxed list?

Reply:
Sure. changed to `IntList`